### ISSUE TITLE
Context filled with empty objects

### ISSUE THREAD

**Issue by josephmcasey** [BohemianCoding/SketchAPI#169]
## Expected Behavior

Given I follow the documentation for the example SVGO Export
And I create a new project in Sketch using File > New From Template > iOS App Icon. 
When I select an Icon and Export Slice as SVG
Then SVG Should be optimized

## Actual Behavior

Given I follow the documentation for the example SVGO Export
And I create a new project in Sketch using File > New From Template > iOS App Icon. 
When I select an Icon and Export Slice as SVG
Then SVG has not been optimized by SVGO

## Steps to Reproduce the Problem

  1. In your home directory, [Follow the SVGO Export Documentation](https://github.com/BohemianCoding/SketchAPI/blob/develop/examples/svgo-export/README.md)
  2. Open Sketch App
  3. Create a new project in Sketch using File > New From Template > iOS App Icon. 
  4. Select bottom left Fox Icon
  5. Change export type from PNG to SVG in bottom right of right panel on window
  6. Select Export in top right of window
  7. Export all slices to home Directory
  8. Inspect exported SVG File

## Specifications

  - Version: OSX 10.13.4
  - Platform: OSX
  - Subsystem: Sketch App v49.3 (51167)

## Observation

It appears that when logging the context object, it is full of empty objects as shown below.

https://github.com/BohemianCoding/SketchAPI/blob/a1c764c67523b5e44877a0bab4e26c9ec8f91abe/examples/svgo-export/src/svgo-export.js#L79

```
export function onExportSlices(context) {
    log(`context: ${JSON.stringify(context)}`)
```

```json
context: {"plugin":{},"action":{},"scriptURL":{},"command":{},"scriptPath":{},"actionContext":{"exports":{},"document":{}}}
```

Consequentially, this causes 

```
const pathToSVGO = context.plugin.urlForResourceNamed(
    'node_modules/.bin/svgo'
  )
```

to evaluate to `null` which means that the bash command for svgo never optimizes the SVG files in the target directory.

**Comment by robintindale** [comment]
Took a look and this isn't a blank context issue - `JSON.stringify` isn't displaying the data properly, if you just do `console.log(context)` you'll see it's working fine.

There are indeed a few issues with this plugin though, looks like it wasn't ported to `skpm` very well. The issues I've found so far http://sketchplugins.com/d/459-svgo-export-is-not-working-on-macos-high-sierra/2 and the `svgo` bin path is incorrect because of the way `skpm` copies the assets.

I'm working on fixing these issues currently 👍

**Comment by mathieudutour** [comment]
to get back to the empty objects in the context, it's because the `toJSON` of a native object is `{}`. With `console.log`, there is an additional layer that prints something nicer.

**Comment by josephmcasey** [comment]
As a new user, I really liked the workable implementation of #170 more than the implementation of #172, which I couldn't get to work.

**Comment by mathieudutour** [comment]
I just fixed a typo in #172, it should be fine now
